engines/dradis-api/lib/dradis/ce/api.rb

```diff
@@ -7,5 +7,6 @@ module API; end
 require 'dradis/ce/api/catch_json_parse_errors'
 require 'dradis/ce/api/routing_constraints'
 require 'dradis/ce/api/version'
-require 'dradis/ce/api/warden_strategy'
+require 'dradis/ce/api/api_token_strategy'
+require 'dradis/ce/api/basic_auth_strategy'
```

Comment:
two `_strategy` suffixes is a smell.

I think that we also have a problem int he main app where we define strategies inline in the initializers when they should live under `./lib/dradis/ce/warden`.

I think the API strategies should live inside the API engine's `./lib/dradis/ce/strategies/api` folder. Then you can remove the `_strategy` suffix.